assets/js/membraneWebRTC.ts

```diff
@@ -51,6 +54,7 @@ export interface SimulcastConfig {
   enabled: boolean;
```

Comment:
While you're at it, the comment for this field seems a little weird, gramatically incorrect almost. I don't think `simulcast` is a verb